site/src/component/Review/SubReview.tsx

```diff
@@ -16,12 +16,19 @@ interface SubReviewProps {
 const SubReview: FC<SubReviewProps> = ({ review, course, professor }) => {
   const [score, setScore] = useState(review.score);
   const [cookies, setCookie] = useCookies(['user']);
+  const [upvoteClassname, setUpvoteClass] = useState("upvote");
+  const [downvoteClassname, setDownvoteClass] = useState("downvote");
 
   const voteReq = async (vote: VoteRequest) => {
     const res = await axios.patch('/reviews/vote', vote);
     return res.data.deltaScore;
   }
 
+  const getColor = async (vote: VoteColorRequest) => {
```

Comment:
Instead of making an API call in each subreview, we should make that 1 API call in Reviews Component and pass the vote status as a prop.